rmw_fastrtps_shared_cpp/QUALITY_DECLARATION.md

```diff
@@ -95,7 +95,19 @@ The latest nightly test results can be seen [here](https://ci.ros2.org/view/nigh
 
 ### Coverage [4.iii]
 
-`rmw_fastrtps_shared_cpp` does not currently track test coverage.
+`rmw_fastrtps_shared_cpp` follows the recommendations for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#coverage), and opts to use branch coverage instead of line coverage.
+
+This includes:
+
+- tracking and reporting line coverage statistics
+- achieving and maintaining a reasonable branch line coverage (90-100%)
+- no lines are manually skipped in coverage calculations
+
+Changes are required to make a best effort to keep or increase coverage before being accepted, but decreases are allowed if properly justified and accepted by reviewers.
+
+Current coverage statistics can be viewed [here](https://ci.ros2.org/job/ci_linux_coverage/lastSuccessfulBuild/cobertura/install_rmw_fastrtps_shared_cpp_include_rmw_fastrtps_shared_cpp/), [here](https://ci.ros2.org/job/ci_linux_coverage/lastSuccessfulBuild/cobertura/src_ros2_rmw_fastrtps_rmw_fastrtps_shared_cpp_include_rmw_fastrtps_shared_cpp/), [here](https://ci.ros2.org/job/ci_linux_coverage/lastSuccessfulBuild/cobertura/src_ros2_rmw_fastrtps_rmw_fastrtps_shared_cpp_src/) and [here](https://ci.ros2.org/job/ci_linux_coverage/lastSuccessfulBuild/cobertura/src_ros2_rmw_fastrtps_rmw_fastrtps_shared_cpp_src_types/). This package does not yet meet the 95% coverage guideline.
```

Comment:
```suggestion
Current coverage statistics can be viewed [here](https://ci.ros2.org/job/nightly_linux_coverage_linux_coverage/lastSuccessfulBuild/cobertura/install_rmw_fastrtps_shared_cpp_include_rmw_fastrtps_shared_cpp/),
[here](https://ci.ros2.org/job/nightly_linux_coverage_linux_coverage/lastSuccessfulBuild/cobertura/src_ros2_rmw_fastrtps_rmw_fastrtps_shared_cpp_include_rmw_fastrtps_shared_cpp/),
[here](https://ci.ros2.org/job/nightly_linux_coverage_linux_coverage_linux_coverage/lastSuccessfulBuild/cobertura/src_ros2_rmw_fastrtps_rmw_fastrtps_shared_cpp_src/) and [here](https://ci.ros2.org/job/nightly_linux_coverage_linux_coverage/lastSuccessfulBuild/cobertura/src_ros2_rmw_fastrtps_rmw_fastrtps_shared_cpp_src_types/).
This package does not yet meet the 95% coverage guideline.
```

Reply:
I think I got the suggestion right in https://github.com/ros2/rmw_fastrtps/pull/404/commits/f0603a6b99affbc4ffe9de843d5a76ca7bf4d383, but PTAL @Blast545 !